securebanking-openbanking-uk-rcs-api/src/test/java/com/forgerock/securebanking/openbanking/uk/rcs/converters/UtilConverter4Test.java

```diff
@@ -29,6 +29,8 @@ public class UtilConverter4Test {
     public static final String DOMESTIC_SCHEDULED_PAYMENT_INTENT_ID = "PDSC_1c214525-d0c8-4d13-xxx-b812c6fafabe";
     public static final String DOMESTIC_STANDING_ORDER_INTENT_ID = "PDSOC_1c214525-d0c8-4d13-xxx-b812c6fafabe";
     public static final String INTERNATIONAL_PAYMENT_INTENT_ID = "PIC_1c214525-d0c8-4d13-xxx-b812c6fafabe";
+    public static final String INTERNATIONAL_SCHEDULED_PAYMENT_INTENT_ID = "PISC_1c214525-d0c8-4d13-xxx-b812c6fafabe";
```

Comment:
Just it's here the constant expected in the above test!

Reply:
I made a mix-up between the Intent Ids for International Scheduled Payments and International Payments.